package.json

```diff
@@ -116,12 +116,10 @@
     "@nextcloud/browserslist-config": "^3.0.0",
     "@nextcloud/eslint-config": "github:nextcloud-libraries/eslint-config#fe67f6b84e3c13eb0ea2478f44939abc2fe7c99c",
     "@nextcloud/stylelint-config": "^2.3.1",
-    "@nextcloud/vite-config": "github:nextcloud-libraries/nextcloud-vite-config#vue3",
+    "@nextcloud/vite-config": "github:nextcloud-libraries/nextcloud-vite-config#10dfa94f6c84c145cf2e10f3c6420b9cd042a3ee",
```

Comment:
You can also keep the branch name, it should then automatically update to the latest commit.
(the exact commit that is used will be shown in package-lock).

Reply:
Yes, I know. I just wanted to be more specific, so it doesn't break in case we rebase `nextcloud-vite-config`. We anyway have to replace it after a final release of `nextcloud-vite-config`.

Reply:
But I can also go back to `vue3`. Should also be fine, unless we rebase and someone has to run `npm install`.

Reply:
No its fine :)

Reply:
We can wait for a final release ;-)